### PR TITLE
DEV2-2899: move the chat-app to external repo

### PR DESCRIPTION
**Pull request by yairco1990**

### CODE REVIEW

**Review by dimacodota — APPROVED** [review]
lgtm, had one small comment regarding the usage of fetch vs.  axios

File: src/tabnineChatWidget/ChatViewProvider.ts [hunk 1]
```diff
@@ -136,61 +132,10 @@ export default class ChatViewProvider implements WebviewViewProvider {
...
-        <div id="root"></div>
-      </body>
-      </html>
-      `;
+  axios.get("http://localhost:3000/index.html")
+    .then((response) => {
+      // eslint-disable-next-line no-param-reassign
+      webviewView.webview.html = response.data;
+    })
+    .catch(console.error);
```
Comment (dimacodota) [inline src/tabnineChatWidget/ChatViewProvider.ts:140]: ill add here a "check that you are running the chat app"  notification or something, just for smoother dev process

File: .github/workflows/alpha_release.yml [hunk 2]
```diff
@@ -36,6 +36,20 @@ jobs:
       with: 
         files: ./src/consts.ts
         replacements: '<INSTRUMENTATION_KEY>=${{ secrets.INSTRUMENTATION_KEY }}'
+    - name: Get Chat Version
```
Comment (dimacodota) [inline .github/workflows/alpha_release.yml:39]: for a later stage, but i think you can define it as a composite action and reuse it 
https://docs.github.com/en/actions/creating-actions/creating-a-composite-action

File: src/tabnineChatWidget/ChatViewProvider.ts (hunk 1 above)
Comment (yairco1990) [reply src/tabnineChatWidget/ChatViewProvider.ts]: great idea
added

**Comment by aarondill** [comment]
From what I can tell, the new repo under `codota/tabnine-chat-app` is currently private. Is there a particular reason for this change, when the assets were previously publicly available under this repo?

**Pull request merged**